PROJECT_PROGRESSION.md

```diff
@@ -105,11 +105,11 @@ To be considered for Growth Stage, the project must meet the At Large requiremen
 ### Impact Stage (formerly 'Top-Level')
 **Definition**
 
-The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintainence, and long-term support. Impact Stage projects are used commonly in enterprise production environments and have large, well-established project communities.     
+The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintenance, and long-term support. Impact Stage projects are used commonly in enterprise production environments and have large, well-established project communities.     
 
 **Examples**
 
-1. Projects that have publicly documented release cycles and plans for LTS.
+1. Projects that have publicly documented release cycles and plans for "Long Term Support" (LTS).
 1. Projects that have themselves become platforms for other projects.
 1. Projects that are able to attract a healthy number of committers on the basis of its production usefulness (not simply 'developer popularity').
```

Comment:
If you're not going to remove the quotation marks above, maybe at least make the instances of quotation marks consistent? You chose double-quotes, but this line uses single-quotes. (I think double-quotes is the better choice, but in any event, let's at least use the same type of quotation marks in both places.)